src/vscode/superbol-vscode-platform/superbol_instance.ml

```diff
@@ -15,10 +15,12 @@
 open Vscode_languageclient
 
 type t = {
+  mutable bundled_superbol : string;
   mutable language_client: LanguageClient.t option
 }
 
 let make () = {
+  bundled_superbol = "superbol-free";
```

Comment:
Please can you comment on the justifications for having this as default?

Reply:
We have many places where `"superbol-free"` is hard-coded. Even if it's the default maybe some of those places are not really useful. For instance, it might be worth just not to call `set_bundled_superbol` from `superbol_vscode_platform.ml` if `find_superbol` fails.

Reply:
Removed that `superbol` by storing the instance in a reference.